docs/content/latest/api/ysql/datatypes/type_datetime/_index.md

```diff
@@ -42,14 +42,58 @@ The <a href="https://www.postgresql.org/docs/11/datatype-datetime.html#DATATYPE-
 {{< /tip >}}
 
-{{< note title="Maximum and minimum supported values" >}}
+{{< note title="Maximum and minimum supported values." >}}
 You might discover that you can define an earlier _timestamp_ value than _4713-01-01 00:00:00 BC_, or a later one than  _294276-01-01 00:00:00_, without error. But you should not rely on this. Rather, you should accept that the values in the "Min" and "Max" columns in the table above specify the _supported_ range.
 
 Notice the "approx" qualifier by the minimum and maximum _interval_ values.You need to understand how an _interval_ value is represented internally as a three-field _[mm, dd, ss]_ tuple to appreciate that the limits must be expressed individually in terms of these fields. The section [_interval_ value limits](./date-time-data-types-semantics/type-interval/interval-limits/) explains all this.
 {{< /note >}}
 
 Modern applications almost always are designed for global deployment. This means that they must accommodate timezones—and that it will be the norm therefore to use the _timestamptz_ data type and not _date_, plain _time_, or plain _timestamp_. Application code will therefore need to be aware of, and to set, the timezone. It's not uncommon to expose the ability to set the timezone to the user so that _date-time_ moments can be shown differently according to the user's present purpose.
 
+## Special date-time manifest constants
+
+PostgreSQL, and therefore YSQL, support the use of several special manifest _text_ constants when they are typecast to specified _date-time_ data types, thus:
+
+| constant    | valid with                         |
+| ----------- | ---------------------------------- |
+| 'epoch'     | date, plain timestamp              |
+| 'infinity'  | date, plain timestamp, timestamptz |
+| '-infinity' | date, plain timestamp              |
+| 'now'       | date, plain time, plain timestamp  |
+| 'today'     | date, plain timestamp              |
+| 'tomorrow'  | date, plain timestamp              |
+| 'yesterday' | date, plain timestamp              |
+| 'allballs'  | plain time                         |
+
+Their meanings are give in section [8.5.1.4. Special Values](https://www.postgresql.org/docs/11/datatype-datetime.html#DATATYPE-DATETIME-SPECIAL-VALUES) in the PostgreSQL documentation.
+
+{{< tip title="Avoid using all of these special constants except for _'infinity'_ and _'-infinity'_." >}}
+
+The implementation of the function [random_test_report_for_modeled_age()](./functions/miscellaneous/function-age/#function-random-test-report-for-modeled-age) shows a common locution where _'infinity'_ and _'-infinity'_ are used to initialize maximum and minimum values for _timestamp_ values that are updated as new _timestamp_ values arise during a loop's execution.
```

Comment:
```suggestion
The implementation of the function [random_test_report_for_modeled_age()](./functions/miscellaneous/age/#function-random-test-report-for-modeled-age) shows a common locution where _'infinity'_ and _'-infinity'_ are used to initialize maximum and minimum values for _timestamp_ values that are updated as new _timestamp_ values arise during a loop's execution.
```

Reply:
That was a broken link . Suggested the right one.

Reply:
Good catch. It's very hard to spot broken links like these when they were once OK and then arrive, in bulk, as a consequence of content re-factoring. We need a locally usable mechanical link-checker. But even that is unlikely to be foolproof because a URL that ends in `#bad-tag` (where `bad-tag` isn't found on the page) simply thrusts you silently to the top of the page.

It's fixed now, of course.